crates/transaction-pool/src/lib.rs

```diff
@@ -203,7 +226,7 @@ where
         transaction: Self::Transaction,
     ) -> PoolResult<TxHash> {
         let (_, tx) = self.validate(origin, transaction).await;
-        self.pool.add_transactions(origin, std::iter::once(tx)).pop().expect("exists; qed")
+        self.pool.add_transactions(origin, std::iter::once(tx.unwrap())).pop().expect("exists; qed")
```

Comment:
no unwraps please.

why is this safe here?

Reply:
so instead I should use pattern matching and handle the Err case ?

Reply:
this unwrap is not safe because this can be an error.

please handle the result.